src/parcel_snoopi_deep.jl

```diff
@@ -24,7 +28,45 @@ function flatten_times(timing::Core.Compiler.Timings.Timing; tmin_secs = 0.0)
         end
         append!(frontier, t.children)
     end
-    return sort(out; by=tl->tl[1])
+    return sort(out; by=first)
+end
+
+"""
+    accumulate_by_method(pairs; tmin_secs = 0.0)
```

Comment:
Also, can we add this as a reference to the docstring on `@snoopi_deep`?:
https://github.com/timholy/SnoopCompile.jl/blob/2f88978f6b7f691b3d46c515f3281d24c7342c15/SnoopCompileCore/src/snoopi_deep.jl#L35-L38

Reply:
Yes. I'm a little unhappy that the docstrings (really, Example sections) I've added are so long. I think I'll refactor that to include an example in `@snoopi_deep` that covers all the use cases and then refer to it (without repeating it) from each individual function. But it will be easiest to merge them all and then fix the docstrings, just so they can be harmonized. So don't worry too much about the Example section for now.